src/components/Cluster/ClusterDetail/ClusterDetailView.js

```diff
@@ -343,7 +331,7 @@ class ClusterDetailView extends React.Component {
                 />
               )}
 
-              {targetRelease && (
+              {canClusterUpgrade && (
```

Comment:
somewhat unrelated. Just felt that we shouldn't render the modal at all if we cant upgrade the cluster anyways.

It used to be that we always could upgrade when we had a next release (which is what targetRelease represents)

But that's no longer the case.